README.md

```diff
@@ -77,6 +77,56 @@ The following points can be customized in a packer variables file `variables.pkr
 - **Provisioning Scripts**: You can replace or modify the provisioning scripts located in the `./files/` directory.
   These scripts and files are copied and executed during the AMI creation process.
 
+## Amazon Linux Image Build
```

Comment:
I would suggest we change this to: "Building an AMI based on Amazon Linux"

Reply:
Done.